docs/development/core/development-functional-tests.asciidoc

```diff
@@ -133,6 +141,35 @@ The `FunctionalTestRunner`'s primary purpose is to execute test files. These fil
 **Test Suite**:::
 A test suite is a collection of tests defined by calling `describe()`, and then populated with tests and setup/teardown hooks by calling `it()`, `before()`, `beforeEach()`, etc. Every test file must define only one top level test suite, and test suites can have as many nested test suites as they like.
 
+**Tags**:::
```

Comment:
Could we rearrange this a bit so the `describe()` bit comes first? For example:

Use tags to group `describe()` functional tests. Tags include:
* `ciGroup{id}` - Assigns test suite to a specific CI worker
* `skipCloud` and `skipFirefox` - Excludes test suite from running on Cloud or Firefox
* `smoke` - Groups tests that run on Chrome and Firefox